src/lanfactory/onnx/transform_onnx.py

```diff
@@ -1,14 +1,14 @@
+"""This module contains the function to transform Torch/Jax models to ONNX format.
```

Comment:
this file operates only on Torch right?

Reply:
took a look on main branch. 
That seems to be the current state of affairs simply. 
I think my memory fooled me then, and the torch side of things has de facto still more importance than intended. 

I will look into expanding the test-suite, since we had anyways already established that we need the torch dependency for the data loaders.